business/data-confidentiality.md

```diff
@@ -0,0 +1,15 @@
+---
+description: Data confidentiality in machine translation
+---
+
+# Data confidentiality
+
+**Data confidentiality** refers to how the machine translation providers use the specific data sent for translation. Providers can generate translations with or without storing the input. Machine translation providers need data to train and improve their translation service.
```

Comment:
> **Data confidentiality** terms or policies govern how machine translation providers use client data.

> Machine translation providers use client data to train and improve their translation systems.  Machine translation clients require data confidentiality for:
> - Compliance with regulations and standards like HIPAA and GDPR
> - Compliance with security reviews
> - Competitiveness

> ### Data types
> - Training data - parallel data
> - Request data - original input

> ### Approaches
> - Client-specific - The data is not used to train models for any other clients.
> - No-trace - The request data is 
> - Auto-delete - The request data is cached, and deleted after a set time period, for example after 72 hours.
> - Delete on request - The training data or request data is deleted upon the client's request.
> - On-premise deployment ("on-prem")
> - On-device models

Reply:
IMHO what's missing here a bit is some way to make it relative to the risks of TMS / CAT / human translators.